htdocs/install/mysql/tables/llx_societe_rib.sql

```diff
@@ -36,6 +37,7 @@ create table llx_societe_rib
   code_banque						varchar(128),  -- bank code
   code_guichet						varchar(6),    -- desk code
   number							varchar(255),  -- account number
+  cci								varchar(32),   -- Interbank code to make a transfer between accounts at different banks
```

Comment:
If you add the field cci into table llx_societe_rib, you must also add it into migration script.
I pushed a PR to do it.